packages/core-mobile/app/services/fcm/AppCheckService.ts

```diff
@@ -0,0 +1,26 @@
+import {
+  firebase,
+  FirebaseAppCheckTypes
+} from '@react-native-firebase/app-check'
+import Logger from 'utils/Logger'
+
+class AppCheckService {
+  verifyAppCheck = async (): Promise<void> => {
+    try {
+      const { token } = await firebase.appCheck().getToken(true)
+
+      if (token.length > 0) {
+        Logger.info('AppCheckService:verifyAppCheck passed')
+      }
+    } catch (error) {
+      Logger.error(
```

Comment:
what happen when verifyAppCheck failed?

Reply:
this function is not used ATM, here only for testing purposes